The Design Rules v1.0.md

```diff
@@ -205,6 +205,15 @@ Although the REST architectural style does not impose a specific protocol, REST
   </table>
 </div>
 
+<div class="rule" id="api-XX">
+  <p class="rulelab"><strong>API-XX</strong>: Handle invalid or illegal parameters</p>
+  <p>The HTTP specification [[rfc7231]] specifies that a code 400 (bad request) must be sent as response to *something that is perceived to be a client error*. Furthermore, the
+  [[OGC API-Features specification]](http://www.opengis.net/doc/IS/ogcapi-features-1/1.0) states that *the server shall respond with a status code 400 if the request URI includes a query parameter that is not specified in the definition* (Requirement 8)</p>
+  <p>A client requesting information using a non-existing parameter will assume that the response is filtered using that parameter. For example, when a user user a non-existing parameter-value pair like *colour=red* the user will assume that only objects are returned with that colour. When a service ignores non-existing parameters it will return *all* objects, regardless of their colour.</>
```

Comment:
```suggestion
  <p>A client requesting information using a non-existing parameter will assume that the response is filtered using that parameter. For example, when a user applies a non-existing parameter-value pair like `colour=red` the user will assume that only objects are returned with that colour. When a service ignores non-existing parameters it will return *all* objects, regardless of their colour.</>
```